styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -330,12 +326,13 @@ public void create(Environment environment) {
         new StateInitializingTrigger(stateFactory, stateManager, storage);
     final TriggerManager triggerManager = new TriggerManager(trigger, time, storage, stats);
 
-    final Scheduler scheduler = new Scheduler(time, timeoutConfig, stateManager, workflowCache,
-                                              storage, trigger);
-
-    final Consumer<Workflow> workflowChangeListener = workflowChanged(workflowCache, storage,
-                                                                      stats, stateManager, time);
+    final WorkflowInitializer workflowInitializer = new WorkflowInitializer(storage, time);
```

Comment:
inline this maybe as it's only used by `workflowChangeListener`? or move it closer to where it is used.